community.html

```diff
@@ -72,25 +72,26 @@ <h2> {{ pres.title }}</h2>
         <div class="list-group-item-container container-fluid hidden">
           <div>
             <p><em>{% if pres.venue %}{{ pres.venue }} • {% endif %}{% if pres.city %}{{pres.city}} • {% endif %}{{ pres.date | date: '%B %Y' }}</em></p>
-            
-            {{ pres.content }}
 
-            <a href="{{ pres.media-url }}">
+            {{ pres.content }}
+            {% for murl in pres.media-url %}
```

Comment:
how about this: a new media-url-list field.  
if it sees a `pres.media-url-list` it does this looping thing, and if it sees a regular `pres.media-url` it does its current logic

Reply:
i think the current version will actually work with lists or single entries, so we should be good